.github/workflows/create-ecr.yml

```
@@ -0,0 +1,49 @@
+name: Create ECR Repository
+
+on:
+    workflow_dispatch:
+
+env:
+    TF_VAR_app_environment: ${{ github.event.inputs.deploy_environment }}
```

actionlint (1.7.4)

property "deploy_environment" is not defined in object type {}

(expression)